.travis.yml

```diff
@@ -39,15 +39,21 @@ env:
 
 before_install:
   - gem install slather --no-rdoc --no-ri --no-document --quiet
+  - npm install -g markdown-link-check
```

Comment:
Moved the `npm install`, and I've googled around a lot with regards to the `cache:` and you can add a bit which is `directories:` so I tried specifying `node_modules` but I couldn't see this effecting anything.

Happy to take input though!